index.html

```diff
@@ -9,7 +9,7 @@
 
   <body id="body" style="text-align: center">
     <h1> FREECODECAMP SURVEY FORM</h1>
-    <h2>Thank you for taking part.</h2>
+    <h1>Thank you for taking part.</h1>
     <div class="con">
```

Comment:
Hi 😀 ? Kindly use one H1 semantic tag for your project as it creates a hierarchy and gives your page a more logical structure. It makes it easy for readers (and those using screen readers) to skim and understand what your page is about.

Reply:
Good job :+1: